### ISSUE TITLE
Integrating with Slack

### ISSUE THREAD

**Issue by pratyushmayank** [dialogflow/dialogflow-fulfillment-nodejs#139]
Hi ,
I am new to the Dialogflow. I am facing the below issue while integrating it with slack .Do we need to enable Webhook before integrating with slack or inline editor(google's cloud function) will work.

As of now I haven't enabled the Webhook and I am trying to integrate it with slack then in the event subscription section I am not getting the response as "Verified". Below is the error for the same.

"Your request URL gave us a 500 error. Update your URL to receive a new request and challenge value."

Please help for the same and thanks in advance.

**Comment by sarahdwyer** [comment]
@pratyushmayank Have you been following the [Slack Integration docs](https://dialogflow.com/docs/integrations/slack)?

**Comment by pratyushmayank** [comment]
@sarahdwyer Yes ,I followed exactly the same steps as mentioned in the slack documentation.

**Comment by mounica12** [comment]
@pratyuskmayank Can you please tel how did you solve the issue as I am also facing the same issue?
